### DISCUSSION TITLE
Unable to connect via NC

### DISCUSSION THREAD

**Discussion by gmaclean1981** [ksheumaker/homeassistant-apsystems_ecur#206] [General]
Hello!

I'm about to get set up with solar and was starting to set up my ECU-R to get it configured for Home Assistant.

When I attempt to NC it however, it won't respond to "APS1100160001END" and will just sit there.

```
/config # nc -v 10.0.2.15 8899
10.0.2.15 (10.0.2.15:8899) open
APS1100160001END
```

My device info:
ECU-R-NA
Article no. 209001
21620012....
Date: 23.7.15

Any thoughts? Thank you!

**Comment by HAEdwin** [comment]
Make absolutely sure you have the correct IP-address specified. If you have other ESP nodes it might easily be the wrong one. Turn off the ECU and see if `ping 10.0.2.15 -t` stops responding.

**Reply by gmaclean1981** [reply]
Sorry, didn’t catch this response, thanks for responding!

Yeah, I’m certain this is the device.

any other thoughts by chance?

**Reply by HAEdwin** [reply]
Is the ECU connected by ethernet cable or wireless? If ethernet cabled try wireless or vice versa. Depending on the ECU model the port 8899 is opened only on the wireless interface.
